config/game_options.txt

```diff
@@ -255,31 +255,35 @@ DEFAULT_LAWS 1
 ## Attempting to upload laws past this point will fail unless the AI is reset
 SILICON_MAX_LAW_AMOUNT 12
 
-## Uncomment to give players the choice of their species before they join the game
+## never comment enables gofast
 JOIN_WITH_MUTANT_RACE
 
 
-## Roundstart Races
+## roundstart fasts
 ##-------------------------------------------------------------------------------------------
-## Uncommenting races will allow them to be choosen at roundstart while join_with_muntant_race is on
+## dont uncomment any they dont gofast
 
-## Disabling humans is possible but a bit buggy. Some code relies on the notion that white people exist in space.
-ROUNDSTART_RACES human
+## gota gofast
+ROUNDSTART_RACES sanic
+ROUNDSTART_RACES sanichu
 
```

Comment:
Configs are not pulled from repo by tg server. Any changes made by you will be lost. Thus, the PR will break.